### PR TITLE
Update `read_csvs()` naming, argument and testing

### LINKED ISSUE #59: [ENH] Naming conventions
I wanted to discuss naming conventions for the various functions and arguments for consistency. `expand_column` has a parameter `column` whereas `add_column` had `col_name`.

Also, is `_column` necessary in each function? Would it be ok to just have an `add()` or `transform()` method? In general I'm more on the side of more verbose function names, but just wanted to throw the idea out there.

Similarly, when following the format of `limit_column_characters` functions like `change_type` should probably be named `change_column_type`.

I'm not married to any of this (except the function arguments, those should be consistent) but wanted to get peoples' ideas.

### PR DESCRIPTION
**Pull request by HectorM14**
# PR Description

Please describe the changes proposed in the pull request: 

- Update naming of arguments in `read_csvs()`
- Accept string or iterable of strings as argument
- Move tests to `io` folder

**This PR resolves #59 .**

# PR Checklist

Please ensure that you have done the following:

1. [X] PR in from a fork off your branch. Do not PR from `<your_username>`:master, but rather from `<your_username>`:<branch_name>.

2. [X] If you're not on the contributors list, add yourself to `AUTHORS.rst`.

## Quick Check

To do a very quick check that everything is correct, follow these steps below:

- [ ] Run the command `make check` from pyjanitor's top-level directory. This will automatically run:
    - [X] black formatting
    - [X] pycodestyle checking
    - [X] running the test suite
    - docs build
    
Once done, please check off the check-box above.
    
If `make check` does not work for you, you can execute the commands listed in the Makefile individually.

## Code Changes

If you are adding code changes, please ensure the following:

- [ ] Ensure that you have added tests.
- [x] Run all tests (`$ pytest .`) locally on your machine.
    - [X] Check to ensure that test coverage covers the lines of code that you have added.
    - [X] Ensure that all tests pass.

# Relevant Reviewers

Please tag maintainers to review.

- @ericmjl

### CODE REVIEW

**Comment by ericmjl** [comment]
Wow, very impressed by your thoroughness here, @HectorM14! I know everything is passing, but I'd like to give this a quick look through before merging.

**Comment by HectorM14** [comment]
Sure thing @ericmjl! I forgot to mention that I updated the generated .csv files when testing to be rectangular to make it more obvious (to me) how the dataframes were being manipulated.

File: janitor/io.py [hunk 1]
```diff
@@ -1,34 +1,49 @@
...
 from glob import glob
+from typing import Iterable, Union
 
 import pandas as pd
 
+from .utils import deprecated_alias
 
-def read_csvs(filespath: str, seperate_df: bool = False, **kwargs):
+
+@deprecated_alias(seperate_df="separate_df", filespath="files_path")
```
Comment (ericmjl) [inline janitor/io.py]: Great catch here on the spelling! I wish there was an automatic way to catch these.

Comment (HectorM14) [reply janitor/io.py]: Sure thing! I find code spell checkers hit or miss with variable names, but I wouldn't be surprised if there is something decent out there already.

**Pull request merged**